### DISCUSSION TITLE
Can you use a lateral join with saved queries?

### DISCUSSION THREAD

**Discussion by Weshag** [getredash/redash#6972] [Q&A]
Using two saved PGSQL queries. Example using query_111 which is start and end (days, weeks, months, your choice) with a lateral join to query_222 counting events based on the periods from query_111. My attempts all seem to fail with an error.

select intervals.start
, intervals.end
, Summary.*
from query_111 intervals
left join lateral
    (select
    sum(case when (created < intervals.end) then 1 else 0 end) as active
    sum(case when (closed > intervals.start) and (closed < intervals.end) then 1 else 0 end) as closed
    from query_222
   )  Summary on true

**Comment by eradman** [comment]
This is a question about writing a query for PostgreSQL, not a question about Redash. Closing

**Comment by RobinDarrah** [comment]
Hi @Weshag,

The query results data source uses SQLite - which doesn't support `LATERAL`

[Query Results Datasource](https://redash.io/help/user-guide/querying/query-results-data-source)
